Pages/products.html

```diff
@@ -0,0 +1,514 @@
+<!DOCTYPE html>
```

Comment:
UX improvement: badge-icons apart in the header and stick the aside bar to the left so that it will not move when we go down to the footer.
![Screenshot 2022-02-20 at 12 40 25 AM](https://user-images.githubusercontent.com/79623546/154815434-405ebce4-288d-49fb-af41-1786f39592ac.png)
![Screenshot 2022-02-20 at 12 40 46 AM](https://user-images.githubusercontent.com/79623546/154815440-5883ca70-0c9f-4d4c-8bd1-124389fec8df.png)

Reply:
Thank you for these suggestions, I have made changes for the filter sidebar and made it fixed. For the badges I will surely do some changes in next iteration as it is related to CL.